test_util/setup.go

```diff
@@ -0,0 +1,69 @@
+package testutil
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/joho/godotenv"
+	"github.com/strangelove-ventures/noble-cctp-relayer/cmd"
+	"github.com/strangelove-ventures/noble-cctp-relayer/ethereum"
+	"github.com/strangelove-ventures/noble-cctp-relayer/noble"
+	"github.com/strangelove-ventures/noble-cctp-relayer/types"
+	"github.com/stretchr/testify/require"
+)
+
+var EnvFile = os.ExpandEnv("$GOPATH/src/github.com/strangelove-ventures/noble-cctp-relayer/.env")
+
+func init() {
+	err := godotenv.Load(EnvFile)
+	if err != nil {
+		fmt.Println(fmt.Errorf("error loading env file"))
+		os.Exit(1)
+	}
+}
+
+func ConfigSetup(t *testing.T) (a *cmd.AppState, registeredDomains map[types.Domain]types.Chain) {
+
+	var testConfig = types.Config{
+		Chains: map[string]types.ChainConfig{
+			"noble": &noble.ChainConfig{
+				ChainID: "grand-1",
+				RPC:     os.Getenv("NOBLE_RPC"),
+			},
+			"ethereum": &ethereum.ChainConfig{
+				ChainID:          11155111,
+				Domain:           types.Domain(0),
+				MinterPrivateKey: "1111111111111111111111111111111111111111111111111111111111111111",
+				RPC:              os.Getenv("SEPOLIA_RPC"),
+				WS:               os.Getenv("SEPOLIA_WS"),
+			},
+		},
+		Circle: types.CircleSettings{
+			AttestationBaseUrl: "https://iris-api-sandbox.circle.com/attestations/",
```

Comment:
I agree there should be a validate method. 
Can you elaborate on what you're thinking?

Only the Circle `attestation-base-url` requries a "/" at the end. So I can check that "/" exists for that field.

For the RPC's and Websockets, what do you think the best way to validate those are?
Maybe use  `url.ParseRequestURI()`?

Reply:
Since this PR is already fairly large, I'm going to break this off into a separate issue. 

I need to think how to integrate this as well. Maybe integrate int InitAppState() and have it return an error. So change all the cobra `PersistentPreRun` back to `PersistentPreRunE`.

Good call out though.